src/components/shared/chip/chip.tsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { Text } from "@lifesg/react-design-system";
+import { ChipButton } from "./chip.styles";
+
+export const Chip = ({ children, ...otherProps }: React.ButtonHTMLAttributes<HTMLButtonElement>) => (
+	<ChipButton type="button" {...otherProps}>
+		<Text.XSmall className="chipText" weight="semibold">
```

Comment:
for classname, do u wanna use kebab-case instead? `chip-text`

Reply:
decided to remove it because the className is not used